AppKit/CPViewController.j

```diff
@@ -176,6 +176,58 @@ var CPViewControllerCachedCibs;
         _view = [CPView new];
 }
 
+- (void)loadViewAsynchronously:(Function/*(view, error)*/)aHandler
```

Comment:
In this case, I would prefer loadViewAsynchronouslyUsingBlock: which is more in line with our current naming pattern for closures.